dev/io.openliberty.cdi.core_fat/fat/src/io/openliberty/cdi/internal/core/RepeatRule.java

```diff
@@ -38,16 +39,16 @@
  */
 public class RepeatRule<T> implements TestRule {
 
-    private T[] repeats;
+    private RepeatTestAction[] repeats;
 
-    private T currentRepeat;
+    private RepeatTestAction currentRepeat;
```

Comment:
Every use of `T` has been removed. The type parameter should have been removed as well and the Javadoc updated.